protocol/messenger_mailserver_cycle.go

```diff
@@ -129,13 +129,9 @@ func (m *Messenger) Storenodes() ([]peer.AddrInfo, error) {
 func (m *Messenger) checkForStorenodeCycleSignals() {
 	defer gocommon.LogOnPanic()
 
-	if m.transport.WakuVersion() != 2 {
-		return
-	}
```

Comment:
Is this no longer needed because the messaging api now explicitly only supports WakuV2

Reply:
Indeed, wakuv1 was completely removed, this is a leftover.